code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -1220,11 +1217,13 @@
 			if(L == user || ishuman(L))
 				continue
 			L.apply_damage(aoe, RED_DAMAGE, null, L.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+			aoe = (aoe / 2) //half damage per target hit
```

Comment:
This will be weird regarding which it decides to pick for the most damage.

Reply:
This is true, but it's a good alternative to lowering the damage globally. I think I will give it a minimum to prevent things from getting to weird.